packages/toml/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@covector/toml",
+  "version": "0.0.0",
+  "files": [
+    "pkg/covector_toml_bg.wasm",
+    "pkg/covector_toml.js",
+    "pkg/covector_toml.d.ts",
+    "dist/index.js",
+    "dist/index.d.ts"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "wasm-pack build --release --no-pack --target nodejs && rimraf pkg/.gitignore",
```

Comment:
I think I would make this a `prebuild`, and then change `build:ts` to `build`. I think this means that we will get a `wasm-pack` prior to other builds without having to change any of our workflows.

Reply:
Ahh, forgot we aren't doing a workspace build at the root. I think on top of this, we also need to add `"prebuild": "npm run prebuild -w @covector/toml",` at the root.